src/index.ts

```diff
@@ -15,3 +15,10 @@ export { mutate } from "swr";
 export { default as useRequest } from "./useRequest";
 export { default as Table } from "./components/Table";
 export { ConnectedTable } from "./components/Table";
+
+export {
+  default as AuthProvider,
+  useToken as useAuthToken,
```

Comment:
We can't export the auth provider token as `useToken` because the alias is already used by `DecodeProvider`exports.